### DISCUSSION TITLE
why it just not work?

### DISCUSSION THREAD

**Discussion by strayshouraki** [UltimaHoarder/UltimaScraper#1746] [Q&A]
![image](https://user-images.githubusercontent.com/83946918/138603604-4930b204-cdfa-46b2-ba58-722dbc7756ed.png)
why 
![image](https://user-images.githubusercontent.com/83946918/138603635-b68ebf85-bf51-42c1-8918-001e06b9f56d.png)

**Comment by JaegerMedia** [comment]
It says you have to run it using Python 3.9

python3.9 start_ofd.py

Make sure you have python version 3.9 installed first.

Honestly this when thing would be easier on Linux.  Install Ubuntu through WSL or a VM or something.
